proto/user.proto

```diff
@@ -7,76 +7,62 @@ syntax = "proto3";
 package typedb.protocol;
 
 message UserManager {
+    message All {
+        message Req {}
 
-    message Contains {
-        message Req {
-            string username = 1;
-        }
         message Res {
-            bool contains = 1;
-        }
-    }
-
-    message Create {
-        message Req {
-            string username = 1;
-            string password = 2;
+            repeated User users = 1;
         }
-        message Res {}
     }
 
-    message Delete {
+    message Contains {
         message Req {
-            string username = 1;
+            string name = 1;
         }
-        message Res {}
-    }
 
-    message All {
-        message Req {}
         message Res {
-            repeated User users = 1;
+            bool contains = 1;
         }
     }
 
-    message PasswordSet {
+    message Get {
         message Req {
-            string username = 1;
-            string password = 2;
+            string name = 1;
+        }
+
+        message Res {
+            User user = 1;
         }
-        message Res {}
     }
 
-    message Get {
+    message Create {
         message Req {
-            string username = 1;
-        }
-        message Res {
             User user = 1;
         }
+
+        message Res {}
     }
 }
 
 message User {
-    string username = 1;
-    optional int64 password_expiry_seconds = 2;
+    string name = 1;
+    string password = 2;
 
-    message PasswordUpdate {
+    message Update {
         message Req {
-            string username = 1;
-            string password_old = 2;
-            string password_new = 3;
+            string name = 1;
+            string password = 2;
+            User user = 3;
```

Comment:
What do we use the `User` for in the request? Do we find the existing user by this message, and are `name` and `password` the updated fields then?